docs/source/user_guide/automl.ipynb

```diff
@@ -645,17 +1598,1393 @@
+      "\n",
+      "Port 8787 is already in use.\n",
+      "Perhaps you already have a cluster running?\n",
+      "Hosting the HTTP server on port 49984 instead\n",
+      "\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Using default limit of max_batches=1.\n",
+      "\n",
+      "Generating pipelines to search over...\n",
+      "2 pipelines ready for search.\n",
+      "\n",
+      "*****************************\n",
+      "* Beginning pipeline search *\n",
+      "*****************************\n",
+      "\n",
+      "Optimizing for Log Loss Binary. \n",
+      "Lower score is better.\n",
+      "\n",
+      "Using DaskEngine to train and score pipelines.\n",
+      "Searching up to 1 batches for a total of 3 pipelines. \n",
+      "Allowed model families: linear_model\n",
+      "\n",
+      "Evaluating Baseline Pipeline: Mode Baseline Binary Classification Pipeline\n",
+      "Mode Baseline Binary Classification Pipeline:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 12.868\n",
+      "\n",
+      "*****************************\n",
+      "* Evaluating Batch Number 1 *\n",
+      "*****************************\n",
+      "\n",
+      "Elastic Net Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n",
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n",
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Logistic Regression Classifier w/ Imputer + Standard Scaler:\n",
+      "\tStarting cross validation\n",
+      "\tFinished cross validation - mean Log Loss Binary: 0.077\n",
+      "\tHigh coefficient of variation (cv >= 0.2) within cross validation scores.\n",
+      "\tLogistic Regression Classifier w/ Imputer + Standard Scaler may not perform as estimated on unseen data.\n",
+      "\n",
+      "Search finished after 00:09            \n",
+      "Best pipeline: Logistic Regression Classifier w/ Imputer + Standard Scaler\n",
+      "Best pipeline Log Loss Binary: 0.076807\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loky-backed parallel loops cannot be called in a multiprocessing, setting n_jobs=1\n"
+     ]
+    }
+   ],
+   "source": [
+    "from dask.distributed import Client, LocalCluster\n",
+    "\n",
+    "from evalml.automl.engine import DaskEngine\n",
+    "\n",
+    "dask_engine_t2 = DaskEngine(Client(LocalCluster(processes=True, n_workers = 2)))\n",
```

Comment:
In the final cell, `t2` is listed as two threads but it's actually two processes right?

![image](https://user-images.githubusercontent.com/41651716/127220402-253006c5-c264-4d50-8471-02bad0147901.png)

Reply:
Good eye